commands/store.go

```diff
@@ -39,8 +43,9 @@ var storeCmd = &cobra.Command{
 	Long:  "Allows browsing and deploying OpenFaaS functions from a store",
 }
 
-func storeList(store string) ([]schema.StoreItem, error) {
-	var results []schema.StoreItem
+func storeList(store, platform string) ([]schema.StoreFunction, error) {
+	var results []schema.StoreFunction
+	var storeData schema.StoreV2
```

Comment:
@stefanprodan how does K8s like to do versioning in packages like this?

We had `Store` (v1) now we have Store v2

Reply:
@stefanprodan ?

Reply:
The version should be in the package name not in the struct see https://github.com/openfaas-incubator/openfaas-operator/tree/master/pkg/apis/openfaas

Reply:
@viveksyngh can you address this? cc @rgee0